its/src/test/java/com/sonar/it/vbnet/UnitTestResultsTest.java

```diff
@@ -20,14 +20,9 @@
 package com.sonar.it.vbnet;
 
 import com.sonar.it.shared.TestUtils;
-import com.sonar.orchestrator.Orchestrator;
-import com.sonar.orchestrator.build.ScannerForMSBuild;
 import java.io.IOException;
-import java.nio.file.Path;
-import org.junit.Before;
-import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.Test;
+
+import org.junit.*;
```

Comment:
@andrei-epure-sonarsource Do we generally use this trick? I haven't seen it elsewhere AFAICT.